Consolidate box-sizing property for checkboxes and radio buttons in html.css

annevk opened the pull request:
#### f35218768146b231ba64439f3da356edd7579937
<pre>
Consolidate box-sizing property for checkboxes and radio buttons in html.css
<a href="https://bugs.webkit.org/show_bug.cgi?id=263070">https://bugs.webkit.org/show_bug.cgi?id=263070</a>
rdar://116858857

Reviewed by Tim Nguyen.

Due to conditional inclusion we set box-sizing three times, but it
appears it&apos;s only needed once. This results in some nice cleanup.

* Source/WebCore/css/html.css:
(input:is([type=&quot;radio&quot;], [type=&quot;checkbox&quot;])):
(#endif):

Canonical link: <a href="https://commits.webkit.org/269254@main">https://commits.webkit.org/269254@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7d6cb38d6a8a543eab893bfca335c8667a8d4e49

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/22028 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/22239 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/23097 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/23911 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/20392 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/22274 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/26488 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/22557 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/21456 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/22259 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/21919 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/19111 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/24764 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/19025 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/19965 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/26221 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/20045 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/20190 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/24088 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/20619 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/17555 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/19970 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/5250 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/24178 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/20569 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Source/WebCore/css/html.css:
```diff
@@ -1033,12 +1029,6 @@ input:is([type="checkbox"], [type="radio"]):checked:disabled {
 }
 #endif
```
Could be good adding a comment indicating the end of the #endif but I guess the rest of the file doesn't do it.

Like: https://searchfox.org/wubkat/search?q=%23endif+%2F%2F&path=&case=false&regexp=false

annevk replied:
I'll leave that for when I want to make more substantive changes.

annevk committed:
Consolidate box-sizing property for checkboxes and radio buttons in html.css
https://bugs.webkit.org/show_bug.cgi?id=263070
rdar://116858857

Reviewed by Tim Nguyen.

Due to conditional inclusion we set box-sizing three times, but it
appears it's only needed once. This results in some nice cleanup.

* Source/WebCore/css/html.css:
(input:is([type="radio"], [type="checkbox"])):
(#endif):

Canonical link: https://commits.webkit.org/269254@main
